handlers/geolocation/geolocation.go

```diff
@@ -160,25 +161,31 @@ func (c *GeolocationHandlersCollection) HandleStreamSource(ctx context.Context,
 
 	latStr := fmt.Sprintf("%f", lat)
 	lonStr := fmt.Sprintf("%f", lon)
+	var errMist error
 	for i := 0; i < streamSourceRetries; i++ {
-		dtscURL, err := c.Balancer.MistUtilLoadSource(context.Background(), payload.StreamName, latStr, lonStr)
-		if err == nil {
+		var dtscURL string
+		dtscURL, errMist = c.Balancer.MistUtilLoadSource(context.Background(), payload.StreamName, latStr, lonStr)
+		if errMist == nil {
 			return c.resolveReplicatedStream(dtscURL, payload.StreamName)
 		}
 
-		glog.Errorf("error querying mist for STREAM_SOURCE: %s", err)
 		pullURL, err := c.getStreamPull(playbackIdFor(payload.StreamName))
 		if err == nil {
 			if pullURL == "" {
 				// not a stream pull
+				glog.V(6).Infof("unable to find STREAM_SOURCE: %s", errMist)
 				return "push://", nil
 			} else {
 				// start stream pull
 				glog.Infof("replying to Mist STREAM_SOURCE with stream pull request=%s response=%s", payload.StreamName, pullURL)
 				return pullURL, nil
 			}
 		}
-		if !errors.Is(err, errLockPull) {
+		if errors.Is(err, errNoStreamSourceForActiveStream) {
+			// stream is active, but STREAM_SOURCE cannot be found
+			glog.Errorf("error querying mist for STREAM_SOURCE: %s", errMist)
```

Comment:
Could we throw the work "active stream" in here please? Just so I can remember which message is which in the future without referring to the code

i.e `error querying mist for active stream STREAM_SOURCE: %s`

Reply:
Ok